Golem.Tools/SampleApp.cs

```diff
@@ -122,7 +122,7 @@ public async Task Run()
 
                 Message = "Payment Initialization";
                 _logger.LogInformation("Initializing payment accounts for: " + Name);
-                await Task.Run(() => Requestor.InitPayment());
+                await Task.Run(async () => await Requestor.InitPayment());
```

Comment:
micronit: doesn't InitPayment() already return Task, cannot we await it already?

Reply:
good point